### PR TITLE
step 10: styling

### PR DESCRIPTION
**Pull request by jchavarri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -901,6 +901,117 @@ username message, while nonexistent usernames like `a---1` will show a server
...
+      id="username-input"
+      value=username
+      onChange={event => onChange(event->React.Event.Form.target##value)}
+      onKeyDown={event => {
+        let enterKey = 13;
+        if (React.Event.Keyboard.keyCode(event) == enterKey) {
+          onEnterKeyDown();
+        };
+      }}
+      placeholder="Enter GitHub username"
```
Comment (feihong) [inline README.md]: Put normal prop above callback props

File: README.md [hunk 2]
```diff
@@ -901,6 +901,117 @@ username message, while nonexistent usernames like `a---1` will show a server
...
+```
+
+Finally, let's update the `src/dune` file to include the CSS file as a runtime
+dependency in the `app` library:
+
+```
+(library
+ (name app)
+ ...
+ (melange.runtime_deps UsernameInput.module.css)
```
Comment (feihong) [inline README.md]:
```suggestion
 (melange.runtime_deps img/camel-fun.jpg UsernameInput.module.css)
```
I think it might be confusing to readers why this change needs to be made, if they haven't read `esbuild.js` and taken a look inside the `_build` directory.

File: README.md [hunk 3]
```diff
@@ -901,6 +901,117 @@ username message, while nonexistent usernames like `a---1` will show a server
...
+`style` element is added to the `index.html` file:
+
+```html
+  <div id="root"></div>
+  <link href="App.css" rel="stylesheet">
+  <script type="module" src="App.mjs"></script>
+```
+### Step 10 completion check
+
+Voilà! When opening the browser, we should now see a more stylish input field,
```
Comment (feihong) [inline README.md]: Maybe I did something wrong, but I had to restart `npm run serve` before it would show the styles. Actually, it seems like any time I make changes to `index.html`, I had to restart the process to see the effects of those changes.

**Review by feihong — APPROVED** [review]
Looks great!

File: README.md (hunk 2 above)
Comment (jchavarri) [reply README.md]: re: the suggested change to add `camel-fun.jpg`, the image was no longer necessary since step 7, "Displaying data". But because we didn't say anything about it, it's only fair to keep it.

I also explained a bit about runtime deps in https://github.com/ahrefs/react-alicante-workshop/pull/8/commits/c0f9656d8c87ab3c75a14814d21c5e00a14ddfce.

File: README.md (hunk 1 above)
Comment (jchavarri) [reply README.md]: https://github.com/ahrefs/react-alicante-workshop/pull/8/commits/c0f9656d8c87ab3c75a14814d21c5e00a14ddfce

File: README.md (hunk 3 above)
Comment (jchavarri) [reply README.md]: https://github.com/ahrefs/react-alicante-workshop/pull/8/commits/464f1eff632f8935326bc5d149fdea2baafc01cd
